<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_logging</class>
 <widget class="QWidget" name="Page_logging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Логирование</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Отображать:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="cb_show_info">
           <property name="text">
            <string>информационные сообщения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_show_debug">
           <property name="text">
            <string>отладочные сообщения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_show_error">
           <property name="text">
            <string>сообщения об ошибке</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_show_trace">
           <property name="text">
            <string>сообщения трассировки</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>178</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_accept">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_show_info</tabstop>
  <tabstop>cb_show_debug</tabstop>
  <tabstop>cb_show_error</tabstop>
  <tabstop>cb_show_trace</tabstop>
  <tabstop>btn_accept</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
